apps/segment/segment.js

```diff
@@ -1169,10 +1186,9 @@ async function showInfo() {
             td.innerHTML = '<button class="btn-del" '+
             'id=removeModel'+modelCount+' type="button"><i class="material-icons"'+
             'style="font-size:16px;">delete_forever</i>Remove Model</button>';
-            document.getElementById('removeModel'+modelCount).addEventListener('click', () => {
+            document.getElementById('removeModel').addEventListener('click', () => {
               deleteModel(name);
             });
-            modelCount+=1;
```

Comment:
These changes need to be reverted too I think .

Reply:
Thanks for pointing out. I have fixed it.